components/dashboard/src/workspaces/EmptyWorkspacesContent.tsx

```diff
@@ -4,78 +4,62 @@
-        const firstScriptTag = document.getElementsByTagName("script")[0];
-        firstScriptTag.parentNode?.insertBefore(tag, firstScriptTag);
-
-        // Create YouTube player when API is ready
-        window.onYouTubeIframeAPIReady = () => {
-            new window.YT.Player("gitpod-video", {
-                events: {
-                    onStateChange: onPlayerStateChange,
-                },
-            });
-        };
-    }, []);
-
-    const onPlayerStateChange = (event: any) => {
-        if (event.data === window.YT.PlayerState.PLAYING) {
-            trackVideoClick("create-new-workspace");
-        }
+    const onPlayerStateChange = () => {
+        trackVideoClick("create-new-workspace");
     };
 
     return (
         <div className="app-container flex flex-col space-y-2">
-            <div className="px-6 mt-16 flex flex-row items-center justify-center space-x-14">
-                <div>
-                    <iframe
-                        id="gitpod-video"
-                        width="535"
-                        height="307"
-                        src="https://www.youtube.com/embed/1ZBN-b2cIB8?enablejsapi=1&modestbranding=1&rel=0&controls=1&showinfo=0&fs=1"
-                        title="YouTube - Gitpod in 120 seconds"
-                        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
-                        allowFullScreen
-                        className="rounded-xl"
-                    ></iframe>
-                </div>
-                <div>
-                    <div className="flex flex-col items-left justify-center h-96 w-96">
-                        <Heading2 className="text-left mb-4 !font-semibold !text-lg">
-                            Create your first workspace
-                        </Heading2>
-                        <Subheading className="text-left max-w-xs">
-                            Write code in your personal development environment that’s running in the cloud
-                        </Subheading>
-                        <span className="flex flex-col space-y-4 w-fit">
-                            <LinkButton
-                                variant="secondary"
-                                className="mt-4 border !border-pk-content-invert-primary text-pk-content-secondary bg-pk-surface-secondary"
-                                href={"/new?showExamples=true"}
-                            >
-                                Try a configured demo repository
-                            </LinkButton>
-                            <LinkButton href={"/new"} className="gap-1.5">
-                                Configure your own repository
-                            </LinkButton>
-                        </span>
-                    </div>
+            <div className="px-6 mt-16 flex flex-row flex-wrap items-center justify-center gap-x-14 gap-y-10 min-h-96 min-w-96">
```

Comment:
instead of `flex-wrap` you could use a breakpoint. i.e. `flex-col md:flex-row`.
That way you can also e.g. do `md:justify-left`  in line 46